indexing/src/test/java/com/google/enterprise/cloudsearch/sdk/indexing/StructuredDataTest.java

```diff
@@ -82,6 +36,16 @@
 import org.mockito.Mock;
 import org.mockito.junit.MockitoJUnitRunner;
 
+import java.io.File;
+import java.io.IOException;
+import java.util.*;
```

Comment:
same here, please refrain from using wildcard imports.  some IDE like intellij has such default setting but can be easily configured.

Reply:
Found that IntelliJ cannot disable the feature entirely, but you can set the threshold to 99 instances before wildcard is applied.

Reply:
fixed and pushed